### DISCUSSION TITLE
Upload greyed out with S3 virtual host style URLs

### DISCUSSION THREAD

**Discussion by fostermi** [iterate-ch/cyberduck#15356] [Q&A]
Using Netapp StorageGrid Object storage, creating a new Bookmark with a URL like `bucket.s3.example.com` and credentials with full-access successfully connects to and lists contents of a bucket. However, the Upload functionality is greyed out. Connecting to this same bucket using the deprecated path-style works as expected.

**To Reproduce**
Create new bookmark
Under "Server" add "bucket.s3.example.com"
Add Access Key and Secret Key
Connect

**Expected behavior**
Bucket contents list and user can drag-n-drop files or right-click and select "Upload".

**Desktop (please complete the following information):**
 - OS: MacOS 13.5
 - Version 8.6.3

**Screenshots**
<img width="522" alt="cyberduck-bookmark" src="https://github.com/iterate-ch/cyberduck/assets/26175388/3fd9841d-5ede-4ce8-b4e3-8eb12351df87">
<img width="985" alt="cyberduck-bucketlist" src="https://github.com/iterate-ch/cyberduck/assets/26175388/d1b77ed3-6237-43f2-907e-66adaefc84e3">
<img width="288" alt="cyberduck-contextmenu" src="https://github.com/iterate-ch/cyberduck/assets/26175388/b0f59151-9de4-4c82-8861-69db08007eca">

**Comment by dkocher** [comment]
I cannot seem to reproduce the problem by using the _S3 (HTTPS)_ connection profile with version _8.7.1_. Please update to the current release.

**Comment by fostermi** [comment]
I've upgraded to 8.7.1 but the issue still persists. I'm using the same credentials as another connection using the path-style format, which does not have an issue.

**Comment by fabiobulgarella** [comment]
Same problem: when using virtual-host style, Cyberduck behaves as if it were not inside the bucket.
